### DISCUSSION TITLE
Hackeable Editor

### DISCUSSION THREAD

**Discussion by firiox** [atom/atom#22113] [General]
Why Atom is hackeable?

**Comment by ThatXliner** [comment]
Check it out:

You get an [init script](https://flight-manual.atom.io/hacking-atom/sections/the-init-file/) which will be executed at the start of Atom. It has access to the [DOM](https://www.w3schools.com/whatis/whatis_htmldom.asp) of the current window and the [atom global](https://flight-manual.atom.io/api/v1.55.0/AtomEnvironment/).

There's the [stylesheet](https://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks) where you can customize Atom with the power of [CSS](https://www.w3.org/Style/CSS/Overview.en.html).

With just those two, that's enough power to control pretty much everything!
